shared/src/main/java/org/apache/myfaces/shared/renderkit/html/CommonPropertyUtils.java

```diff
@@ -531,11 +531,7 @@ public static void renderLabelPassthroughPropertiesWithoutEvents(ResponseWriter
             long commonPropertiesMarked, UIComponent component) 
     throws IOException
     {
-        if ((commonPropertiesMarked & CommonPropertyConstants.ACCESSKEY_PROP) != 0)
-        {
-            HtmlRendererUtils.renderHTMLStringAttribute(writer, component,
-                    HTML.ACCESSKEY_ATTR, HTML.ACCESSKEY_ATTR);
-        }
+        renderAccesskeyTabindexProperties(writer, commonPropertiesMarked, component);
```

Comment:
ACCESSKEY_PROP is rendered within renderAccesskeyTabindexProperties, so the only addition is the tabindex